activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -285,6 +301,12 @@ def utc_offset
       end
     end
 
+    # Returns the offset of this time zone from UTC in seconds,
+    # taking DST into account at the given time.
+    def utc_total_offset(time=Time.now, dst=true)
```

Comment:
Yes, it should :)

Den søndag den 15. marts 2015 skrev Yasyf Mohamedali <
notifications@github.com>:

> In activesupport/lib/active_support/values/time_zone.rb
> <https://github.com/rails/rails/pull/19279#discussion_r26446679>:
>
> > @@ -285,6 +301,12 @@ def utc_offset
> >        end
> >      end
> >
> > +    # Returns the offset of this time zone from UTC in seconds,
> > +    # taking DST into account at the given time.
> > +    def utc_total_offset(time=Time.now, dst=true)
>
> @kaspth <https://github.com/kaspth> thanks for the suggestion. What about
> #from_offset, should that also follow the same convention?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/19279/files#r26446679>.
>